descriptors/ModuleDescriptor-template.json

```diff
@@ -31,6 +31,31 @@
           "methods": ["DELETE"],
           "pathPattern": "/inn-reach/central-servers/{centralServerId}",
           "permissionsRequired": ["inn-reach.central-servers.item.delete"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/locations",
```

Comment:
we agreed to use `inn-reach` as root element of URI
please update the mapping

Reply:
In addition to remarkabout updating route path of "locations" by preceding with modules name space as @dmtkachenko wrote, please check permissions groups. It seems we need to add here a group of permissions like "inn-reach.locations.all" like it was done for "central servers". Beside that don't forgot to place all the new permission in the "permissionSets" sections with adding "inn-reach.locations.all" to root permissions group "inn-reach.all". All these will allows to add permissions to database during module update.